doc/source/theory/multiphases/cfd/cfd.rst

```diff
@@ -0,0 +1,12 @@
+================================
+Phase Separation Models
```

Comment:
I'm not sure about the name "Phase Separation Models". It seems leading to _phases regrouping on themselves_ (mixed to unmixed).

Both VOF and Cahn-Hiliard (`phase-field`) are under the "One-Fluid Models" ( see [Mirjalili et al., 2017](http://web.stanford.edu/~sjsuresh/mirjalili2017.pdf)); we only solve one set of Navier-Stokes equations to simulate both fluids. 
They are also both interface-capturing methods... but that would maybe be a little too specific. 

```suggestion
One-Fluid Models
```

Reply:
If it is in the Multiphase flow section, I think Amishga is right to use One-Fluid Models